src/course/handbook/projects.md

```diff
@@ -1,75 +1,71 @@
-# Project
+# Projects
 
 Each week, you will be split into teams of 4 for weekly projects. These teams will be change throughout the course, giving you a chance to work with every other member of your cohort.
 
-The teams stay together for the following weeks:
+The founders and coders learning environment is all about collaboration, and this is a great opportunity to practice your workflow in a development team.
```

Comment:
```suggestion
The Founders and Coders learning environment is all about collaboration, and this is a great opportunity to practice your workflow in a development team.
```

Reply:
The pedant in me wants to point out that _practise_ is the UK spelling when using the word as a verb. Although, I'm not sure we've been consistent in the past 🤓